guides/upstreamize.rb

```diff
@@ -42,7 +42,11 @@ def putsv(str)
             line.gsub!(/Capsule( |\u{00A0}|{nbsp})Server/, '{SmartProxyServer}')
             line.gsub!(/Capsules/, '{SmartProxies}')
             line.gsub!(/Capsule/, '{SmartProxy}')
-            line.gsub!(/satellite.example.com/, '{foreman-example-com}')
+            line.gsub!(/satellite\.example\.com/, '{foreman-example-com}')
+            line.gsub!(/capsule\.example\.com/, '{smartproxy-example-com}')
```

Comment:
@lzap, I need your help with those two lines 45, 46. The way it is now replacement does not work. Escaping dots with backslashes did not help too. Do you know how to tell it to replace satellite.example.com? Thanks!

Reply:
What you mean by "it does not work" exactly?

```
$ irb
>> line = "Test for satellite.example.com"
=> "Test for satellite.example.com"
>> line.gsub!(/satellite\.example\.com/, '{foreman-example-com}')
=> "Test for {foreman-example-com}"
```

Reply:
When I run the script it does not pick satellite.example.com, does not replace it. And escaping with backslashed did not help. I see  structures like `|\u{00A0}` above, maybe you know how to escape it properly.